docs/Using-Pantheon/Events-and-Logs.md

```diff
@@ -0,0 +1,87 @@
+# Events and Logs
+
+When a transaction is mined, smart contracts emit events and write logs to the blockchain.  
+
+Logs are associated with the contract address and included in the blockchain but logs are not accessible 
+from within contracts. For example, logs can be used to display all transfers made using a specific contract 
```

Comment:
-> [you can use logs] to display all transfers made using a specific contract[,]

Reply:
Done